Scheduler/observing_blocks/survey-spec-pole-with-rotation.json

```diff
@@ -0,0 +1,140 @@
+{
+    "name": "Spectroscopy",
+    "program": "spec_pole_with_rotation",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "auxtel/latiss_acquire.py",
+            "standard": false,
+            "parameters": {
+                "object_ra": "$ra",
+                "object_dec": "$dec",
+                "object_name": "$name",
+                "acq_filter": "empty_1",
+                "acq_grating": "holo4_003",
+                "acq_exposure_time": 5.0,
+                "program": "spec",
+                "reason": "Generic Survey",
+                "estimated_slew_time": "$estimated_slew_time",
+                "time_on_target": 300.0
+            }
+        },
+        {
+            "name": "auxtel/latiss_take_sequence.py",
+            "standard": true,
+            "parameters": {
+                "object_ra": "$ra",
```

Comment:
`latiss_take_sequence` does not take object coordinates! Here and in all other occurrences.